srcpkgs/qodem/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'qodem'
+pkgname=qodem
+version=1.0.1
+revision=1
+build_style=gnu-makefile
+make_build_args="-f ./build/Makefile.generic"
+make_build_target="clean qodem"
```

Comment:
the source has a configure script in it, so we can use the `gnu-configure` build style instead
```suggestion
build_style=gnu-configure
```